mlflow/sagemaker/__init__.py

```diff
@@ -432,6 +435,8 @@ def _deploy(role, image_url, app_name, model_s3_path, run_id, region_name, mode,
                     will be preserved. If False, these resources will be deleted.
     :param instance_type: The type of SageMaker ML instance on which to deploy the model.
     :param instance_count: The number of SageMaker ML instances on which to deploy the model.
+    :param vpc_config: A dictionary specifying the VPC configuration to use when creating the
+                       new SageMaker model associated with this application.
```

Comment:
Can you give an example of this, since some of the entries are supposed to be arrays of strings?

Reply:
Added an example!